server/settings/tools.py

```diff
@@ -70,7 +72,9 @@ class CreatorToolModel(BaseSettingsModel):
 
     filter_creator_profiles: list[FilterCreatorProfile] = SettingsField(
         default_factory=list,
-        title="Filter creator profiles"
+        title="Filter creator profiles",
+        description="White list of creator labels that will be only shown if "
```

Comment:
_Whitelist_ is one word.
```suggestion
        description="Whitelist of creator labels that will be only shown if "
```

Deadline a few years back changed "whitelist" to "Allow list" due to connotation with with and black in whitelist and blacklist. Do we want to do the same here?
```suggestion
        description="Allowed list of creator labels that will be only shown if "
```